test-rv32emu/device/rv32i_m/M

```diff
@@ -0,0 +1 @@
+I
```

Comment:
Why?

Reply:
Because the file is the same as the file rv32i_m/I, so i used symbolic link.